htdocs/comm/propal/list.php

```diff
@@ -696,6 +696,11 @@
 $reshook = $hookmanager->executeHooks('printFieldListWhere', $parameters); // Note that $action and $object may have been modified by hook
 $sql .= $hookmanager->resPrint;
 
+// Add HAVING from hooks
+$parameters = array();
+$reshook = $hookmanager->executeHooks('printFieldListHaving', $parameters, $object); // Note that $action and $object may have been modified by hook
+$sql .= !empty($hookmanager->resPrint) ? (' HAVING 1=1 ' . $hookmanager->resPrint) : '';
```

Comment:
Are you sure it works on PGSQL ?

Reply:
I think so ;)
https://www.postgresqltutorial.com/postgresql-having/

good to see you

Reply:
@eldy Can you review this hook ?
I think it's important to uniformize and have every available hooks for each type of view (cards and lists)